AIDevGallery/Samples/WCRAPIs/BackgroundRemover.xaml.cs

```diff
@@ -32,10 +32,6 @@ namespace AIDevGallery.Samples.WCRAPIs;
     Model1Types = [ModelType.BackgroundRemover],
     Scenario = ScenarioType.ImageBackgroundRemover,
     Id = "79eca6f0-3092-4b6f-9a81-94a2aff22559",
-    SharedCode = [
-        SharedCodeEnum.WcrModelDownloaderCs,
-        SharedCodeEnum.WcrModelDownloaderXaml
-    ],
     AssetFilenames = [
         "pose_default.png"
```

Comment:
If we are not loading the images anymore, we should remove them from the exported samples.

Reply:
i'll set the image so we are loading it